modules/video/test/ocl/test_optflowpyrlk.cpp

```diff
@@ -121,13 +124,24 @@ OCL_TEST_P(PyrLKOpticalFlow, Mat)
             float errdiff = 0.0f;
 
             if (!eq || errdiff > 1e-1)
+            {
                 ++mistmatch;
+                continue;
+            }
+
+            eq = std::abs(cpuErr[i] - err[i]) < 0.01;
+            if(!eq &&
+               winSize.width >= 16 && winSize.height >=16 &&
+               winSize.width <=24 && winSize.height <=24)
```

Comment:
This test is intended to test optical flow implementation (any "optimized" version with passed parameters).
So this `winSize` limitation looks illogical here.
Especially, these `winSize` values are not covered by test parameters [below](https://github.com/opencv/opencv/pull/8609/files#diff-12bd0224b222e84bc79818257f400787R149)

BTW, Please ignore "OCL_" prefix here - it is for legacy reasons here (to compare with OpenCV 2.4 tests). It doesn't force OpenCL code.